change/@microsoft-fast-components-f057933c-e206-4ff8-ac2a-a7fcde8beb70.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: selected option is not visible when dropdown is opened in fast-select",
```

nit - not necessary to change, but we technically don't need to prefix these descriptions :)
```suggestion
  "comment": "selected option is not visible when dropdown is opened in fast-select",
```